client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/dynamic-settings-form.js

```diff
@@ -170,6 +179,7 @@ const mapDispatchToProps = ( dispatch ) => ({
 DynamicCarrierAccountSettingsForm.propTypes = {
 	carrierType: PropTypes.string,
 	carrierName: PropTypes.string,
+	carrierDesc: PropTypes.string,
```

Comment:
Looks like this is still calling it `carrierDesc` instead of `carrierDescription`

Reply:
Thanks for catching that! Done.